bulkimport_scripts/add_users_for_bulk_import/README.md

```diff
@@ -0,0 +1,34 @@
+# Add Users For Bulk Import Script
+
+The `/bulk-import/users` POST API endpoint in SuperTokens Core allows to add users to the database to bulk import the users. However, the API only allows importing 10,000 users at once. This script can take a JSON file containing a large number of users and call the API in batches of 10,000.
+
+## How to Run
+
+1. Ensure you have Node.js (v16 or higher) installed on your system.
+2. Open a terminal window and navigate to the directory where the script is located.
+3. Run `npm install` to install necessary dependencies.
+4. Run the script using the following command:
+
+    ```
+    node index.js --core-endpoint <CoreAPIURL> --input-file <InputFileName> [--invalid-schema-file <InvalidSchemaFile>] [--remaining-users-file <RemainingUsersFile>]
```

Comment:
If you are going to use the package.json, would it make sense to add an npm script?

Reply:
also, it might be good to show prompts to take these inputs, if not passed via args. what do you think?
for eg. if user didn't pass --core-endpoint, while running the script ask `Enter the core endpoint:` and take the input

Reply:
It will make the script more complex. I think the current UX Is good enough.